### PR TITLE
Add qhelpgenerator-qt5, qdoc-qt5 and qtattributionsscanner-qt5 in INS…

### PR DESCRIPTION
**Pull request by 53845714nF**
Add qhelpgenerator-qt5, qdoc-qt5 and qtattributionsscanner-qt5 in INSTALL.md

Needed in Debian 12 installation

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ i.e. when using distro provided Qt make sure you have (or the equivalent for Qt6
 
 - Redhat,Fedora: qt5-qtbase-private-devel
 - Debian,Ubuntu: qtbase5-private-dev qtdeclarative5-private-dev
+qtattributionsscanner-qt5 qhelpgenerator-qt5 qdoc-qt5
```
Comment (Waqar144) [inline INSTALL.md:24]: shouldn't this be appended to the previous line?

Comment (53845714nF) [reply INSTALL.md:24]: No Markdownlint complains that the line is otherwise too long.

Comment (53845714nF) [reply INSTALL.md]: Oh, I noticed that the linter was not yet happy. I have added a space.

Comment (Waqar144) [reply INSTALL.md]: Looks ok in the preview.

**Pull request merged**